src/main/resources/weka-clustering-mysql-handler.template

```diff
@@ -189,12 +189,7 @@
             pValueCounts = 1.0;
         }
         boolean passedStatClust = pValueCounts>0.05;
-        double pValueKS;
-        if( deviationsScores>0 ) {
-            pValueKS = TestUtils.kolmogorovSmirnovTest(expectedMorphedDistributions, morphedDistributions);
-        } else {
-            pValueKS = 1.0;
-        }
+        double pValueKS = 0.0;

```

Comment:
This looks wrong, the result will then be trivial. Is this not needed anymore, because this is the score-matching, which does not work for clustering?

Do you think this may work in the future and are leaving this as placeholder? If yes, then add a TODO comment instead and clean up the code.

Reply:
Yes, this is the score-matching, which currently does not work for clustering. The purpose of setting this is to merge the results into the same database table as the classification results. Alternatively, the results can be merged by not setting any value for the score-matching in the database table, which would result in the corresponding column being NULL for clustering algorithms per default. 
I will remove the variable and clean up the code.

Reply:
In that case, a TODO sounds more reasonable, that says that there currently is now score-based matching, but that this may be added for soft clustering and this is, currently, as placeholder.